pkg/hostagent/setup.go

```diff
@@ -112,6 +114,11 @@ func (*ClientRPC) SetupVeth(args *SetupVethArgs, result *SetupVethResult) error
 
 		result.HostVethName = hostVeth.Name
 
+		// Force a consistent MAC address based on the IP address
+		if err := ip.SetHWAddrByIP(args.IfName, args.PodIp, nil); err != nil {
```

Comment:
can you check for ipv6? for now, ignore calling if it is of type V6,  as  API is not supporting V6 we can ignore that, next time support comes it is easy to identify, also put the comment there

Reply:
Ok I will handle the ipv6 case.